openhtf/util/multicast.py

```diff
@@ -144,6 +145,16 @@ def send(query,
   # Set up the socket as a UDP Multicast socket with the given timeout.
   sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
   sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, ttl)
+  if local_only:
+    sock.setsockopt(
+        socket.IPPROTO_IP,
+        socket.IP_MULTICAST_IF,
+        # IP_MULTICAST_IF is the 8-byte group address followed by the IP
+        # assigned to the interface on which to listen.
+        struct.pack(
+            '!4sL',
```

Comment:
127.0.0.1 is the universal address of localhost. Or are you asking about '!4sL', which is defined by the struct module's documentation